README.md

```diff
@@ -129,6 +129,12 @@ To run tests, run the following command:
   bin/rails test test/models/article_test.rb
 ```
 
+To run tests on the views, run the following command:
+
+```sh
+  bin/rails bundle exec rspec spec/views
+```
+
```

Comment:
- [ ] I can see that you added all the necessary tests and they should pass, but unfortunately you have some problems with gemfile and gems therefore I am unable to run rspec, kindly fix it:
![image](https://github.com/Dadadon/blog/assets/62705947/7afb5a4d-f434-42d5-a70e-ad835e9853b0)
![image](https://github.com/Dadadon/blog/assets/62705947/a6752f0a-0d89-46dc-89e9-d4b4669434b8)